wpilibj/src/test/java/edu/wpi/first/wpilibj/simulation/SingleJointedArmSimTest.java

```diff
@@ -14,20 +14,13 @@
 class SingleJointedArmSimTest {
   SingleJointedArmSim m_sim =
       new SingleJointedArmSim(
-          DCMotor.getVex775Pro(2),
-          100,
-          3.0,
-          Units.inchesToMeters(19.0 / 2.0),
-          -Math.PI,
-          0.0,
-          10.0 / 2.2,
-          true);
+          DCMotor.getVex775Pro(2), 300, 3.0, Units.inchesToMeters(30.0), -Math.PI, 0.0, true);
 
   @Test
   void testArmDisabled() {
     m_sim.setState(VecBuilder.fill(0.0, 0.0));
 
-    for (int i = 0; i < 12 / 0.02; i++) {
+    for (int i = 0; i < 10 / 0.02; i++) {
```

Comment:
Why was this changed? If it was intentional, the C++ test should be modified to match.

Reply:
Seems like neither matched the other, at all? Just pushed a change to make both tests match